app/src/main/java/com/epfl/beatlink/ui/search/components/GradientTitle.kt

```diff
@@ -0,0 +1,55 @@
+package com.epfl.beatlink.ui.search.components
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.outlined.KeyboardArrowRight
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawWithCache
+import androidx.compose.ui.graphics.BlendMode
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.epfl.beatlink.ui.theme.PrimaryGradientBrush
+
+@Composable
+fun GradientTitle(title: String, button: Boolean, onClick: () -> Unit) {
```

Comment:
i suggest renaming the variable `button` because it is not very clear what it is used for and  why it is a `Boolean`